content/docs/reference/quality-guidelines.md

```diff
@@ -15,4 +15,21 @@ All PRs:
 New addons:
 - must have clear, concise, and easy-to-understand titles, descriptions, info notices (if any), and setting names (if any). English is required: the base addon manifest is used as a reference for our translators. The developers and members of the community should scrutinize and help with this. An effective meter for this is to test it **blindly** -- that is, download and try out the addon before reading the PR description.
 - must be easy to use. The labels and design language used in any UI that the addon adds or modifies should be understandable by whoever will be using that addon. Again, testing blindly is an effective meter for this. Addons without an effective UI communicating its changes will be ranked lower than other addons in the settings page.
-- must be compatible with our existing addons. This doesn't just mean avoiding bugs: addons should complement each other by supporting each other's features. For example, an addon that has a user interface should support our respective dark mode addon.
\ No newline at end of file
+- must be compatible with our existing addons. This doesn't just mean avoiding bugs: addons should complement each other by supporting each other's features. For example, an addon that has a user interface should support our respective dark mode addon.
+
+In order for an approval to be valid, it must have the following checklist accompanying it as a comment:
+```markdown
+<!-- The following checklist items apply to all PRs. -->
+[] Valid reason to merge
+[] To the best of my knowledge, supports all browsers supported by the extension
+[] I personally tested all features in both a Chromium-based browser and Firefox
+[] I personally reviewed all code line-by-line
+<!-- For new addons and features, the following checklist items must also be included and completed. -->
+[] All titles, descriptions, info notices, and setting names provided are clear and concise
+[] I personally found the addon to be easy to use during my testing
+[] I personally tested with all other addons and features enabled and did not find compatibility issues
```

Comment:
Sometimes issues only occur when a particular feature is *not* enabled (because the addon author was using it and never tested the addon without it). Is that worth mentioning here?

Reply:
Probably -- most of these kinds of issues can probably be caught by testing without any other addons enabled.